assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java

```diff
@@ -11742,6 +11707,62 @@ else if (typeOfGrade == Assignment.CHECK_GRADE_TYPE) {
 				}
 			}
 		}
+		
+		
+		public String getGradeForUserInGradeBook(String userId)
+		{
+			return this.getGradeFromGradebook(userId);
+		}
+		
+		protected String getGradeFromGradebook(String userId)
```

Comment:
Thanks Steve. Ok, you're all right, It was due to the previews development I did. I will leave only one method.